lib/src/builder_implementation.dart

```diff
@@ -718,8 +726,16 @@ class _ReflectorDomain {
       String defaultValueCode = "";
       if (parameterNode is DefaultFormalParameter &&
           parameterNode.defaultValue != null) {
-        defaultValueCode =
-            ": ${_extractConstantCode(parameterNode.defaultValue, importCollector, _generatedLibraryId, _resolver)}";
+        var constantCode = _extractConstantCode(parameterNode.defaultValue,
```

Comment:
This is a lot of very similar code - worth extracting a helper?

Reply:
I thought about exactly the same thing, but a quick check made me think they differed here and there. I've checked more precisely now, and moved a 17 line block that occurs twice into a local function.